node/openrpc.go

```diff
@@ -56,7 +56,12 @@ var sharedMetaRegisterer = &go_openrpc_reflect.MetaT{
 		return info
 	},
 	GetExternalDocsFn: func() (exdocs *meta_schema.ExternalDocumentationObject) {
-		return nil // FIXME
+		exdocs = &meta_schema.ExternalDocumentationObject{}
+		description := "ETC Labs Documentation"
+		exdocs.Description = (*meta_schema.ExternalDocumentationObjectDescription)(&description)
+		url := "https://etclabscore.github.io/core-geth/"
```

Comment:
@meowsbits can you please let me know of the URL which will be used for serving the discovery document in our docs website?

Reply:
Thanks for tagging this on; it'll be nice to have set.

How about let's leave the link as-is for now, and then #306 can be responsible for updating it when the precise URI gets firmed up.

Reply:
Sounds perfect to me. Please review and merge this PR then